frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -612,12 +638,15 @@ function _getUnrenderableLayersForCurrentZoom(state: OxalisState) {
         return resolutionInfo.hasIndex(fallbackZoomStep);
       });
     })
-    .map<DataLayerType>(({ layer }) => layer);
+    .map<UnrenderableLayersInfos>(({ layer, resolutionInfo }) => {
+      const smallerOrHigherInfo = resolutionInfo.hasSmallerAndOrHigherIndex(zoomStep);
+      return { layer, smallerOrHigherInfo };
+    });
   return unrenderableLayers;
 }
 
-export const getUnrenderableLayersForCurrentZoom = reuseInstanceOnEquality(
-  _getUnrenderableLayersForCurrentZoom,
+export const getUnrenderableLayersInfoForCurrentZoom = reuseInstanceOnEquality(
+  _getUnrenderableLayersInfosForCurrentZoom,
```

Comment:
two things: 1) the names are not identical (`info` vs `infos`) and 2) I would use a third variant: `_getUnrenderableLayerInfosForCurrentZoom`. The reasoning is that this implies that one gets multiple "layer info"-things. for example, the "LayersInfo` naming sounds like one would only get one "info"-thing which conveys the wrong semantics in my opinion. feel free to disagree, though :D

Reply:
>1) the names are not identical (info vs infos)

:man_facepalming: 

> for example, the "LayersInfo` naming sounds like one would only get one "info"-thing which conveys the wrong semantics in my opinion.

I agree, lets go with `_getUnrenderableLayerInfosForCurrentZoom`